kernel/src/riscv64_qemuvirt_main.rs

```diff
@@ -74,7 +74,7 @@ extern "C" fn k_main(_core_id: usize, device_tree_ptr: usize) -> ! {
 
     kprintln!("[OK] Setup virtual memory");
 
-    let mut arch_interrupts = InterruptsImpl{};
+    let mut arch_interrupts = InterruptsImpl {};
```

Comment:
```suggestion
    let mut arch_interrupts = InterruptsImpl;
```

We can have empty structs, people! Stop putting extra `{}` at the end of them :D This isn't C!

And yeah, that's not relevant to this PR and the comment would be better adressed on the struct definition itself. But still!